src/ControlSurface/Controls/Slider/ScoreSliders.qml

```diff
@@ -109,12 +127,20 @@ ColumnLayout {
                         // Int Slider
                         tmpValue = s.Value.Int
                         break
+                    case Uuid.logFloatSliderUUID:
+                        // LogFloat Slider
+                        tmpValue = s.Value.Float
+                        break
                     default:
                         return
                     }
+
+
+                    /*
                     sliderListModel.set(i, {
```

Comment:
what is this part for ?

Reply:
it is for handling with message from score, but I removed it temporarily because it was broken